maquetado/scss/login.scss

```
@@ -0,0 +1,95 @@
+@import "colors";
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.login-container {
+  background-color: $light-grey;
+  border-top: 8px solid $light-blue;
+  display: flex;
+  height: 400px;
+  justify-content: center;
+  margin-top: 70px;
+  width: 300px;
+}
+
+.login-inputs-container {
+  display: flex;
+  flex-direction: column;
+  &::before {
+    align-self: center;
+    background: url("../maquetado/image/logoWolox.png") no-repeat center/contain;
+    content: "";
+    height: 70px;
+    margin-top: 20px;
+    width: 200px;
+  }
+}
+
+.label {
+  color: $black;
+  font-family: Roboto;
+  font-size: 16px;
+  height: 22px;
+  line-height: 24px;
+  margin-top: 10px;
+  margin-bottom: 5px;
+  width: 41px;
+}
+
+.input {
+  background-color: $white;
+  border-radius: 5px;
+  height: 32px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 252px;
+}
+
+.login-buttons-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.primary-button {
+  background-color: $green;
+  border-radius: 5px;
+  height: 32px;
+  margin-top: 10px;
+  margin-bottom: 25px;
+  position: relative;
+  width: 252px;
+  &::after {
+    background-color: $grey;
+    content: "";
+    height: 2px;
+    left: 0;
+    position: absolute;
+    top: calc(130%);
```

If you are going to use more than 100% percent then do this: 

```suggestion
    top: calc(100% + some pixels);
```